modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -4,64 +4,102 @@ import caseapp._
 import coursier.env.{EnvironmentUpdate, ProfileUpdater}
 
 import scala.io.StdIn.readLine
-import scala.util.Properties
+import scala.util.{Properties, Try}
 
 object InstallHome extends ScalaCommand[InstallHomeOptions] {
   override def hidden: Boolean = true
-  def run(options: InstallHomeOptions, args: RemainingArgs): Unit = {
 
-    val binDirPath      = scala.build.Directories.default().binRepoDir
-    val scalaCliBinPath = binDirPath / "scala-cli"
-
-    if (os.exists(scalaCliBinPath)) {
-      if (options.force) os.remove.all(scalaCliBinPath)
-      else if (coursier.paths.Util.useAnsiOutput()) {
-        println(
-          "scala-cli already exists. Do you want to override it [Y/n]"
-        )
-        val replace = readLine()
-        if (replace == "Y") {
-          os.remove.all(scalaCliBinPath)
-        }
-        else {
-          System.err.println("Abort")
-          sys.exit(1)
-        }
-      }
-      else {
-        System.err.println(
-          s"Error: scala-cli already exists. Pass -f or --force to force erasing it."
-        )
+  private def isOutOfDate(newVersion: String, oldVersion: String): Boolean = {
+    import coursier.core.Version
+
+    Version(newVersion) > Version(oldVersion)
+  }
+
+  private def loggingEqual(version: String) = {
```

Comment:
NIT `log` (verb) sounds better than `logging` (name)
```suggestion
  private def logEqual(version: String) = {
```